### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by corylown** [comment]
Blocked until the following are merged:
https://github.com/sul-dlss/sul-embed/pull/1431
https://github.com/sul-dlss/sul-embed/pull/1420
https://github.com/sul-dlss/sul-embed/pull/1413

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**
